### PR TITLE
The journey to small, cross-functional, Agile teams

### PR DESCRIPTION
**Pull request by jodiedoubleday**
### Blog - "The journey to small, cross-functional, Agile teams"
A quick blog on our experience of transitioning the web team in 2014 to pods. Would love someone to review the post so we can publish
*PR on behalf of Anna Divers*
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by jodiedoubleday** [comment]
testing slack intergration

**Comment by jodiedoubleday** [comment]
Testing slack intergration

**Comment by jodiedoubleday** [comment]
Testing slack integration (with icon?)

**Comment by alannamooney** [comment]
I'm doing one of the editor reviews now

**Comment by alannamooney** [comment]
I'm taking one of the editor reviews.

File: _drafts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-comments: "true"
-author: Anna Divers
-tags: "Product-Owner Lean  Agile Product-Development"
-categories: productowner
-time: 8 mins
----
-
-## A Year in the Life of a Holiday Extras Product Owner and my recommended events for all you PO's out there!
-
-**2014** - **a year of travelling, learning and transforming**.  My New Years Resolution last year was to get out of the building more, learn more from external sources after the previous year very much head down on a conversion and traffic-migration mission.  Here’s how it turned out...
```
Comment (alannamooney) [inline _drafts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Years should be Year's

**Comment by jackdcrawford** [comment]
I can also take an editor review

File: _drafts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 2]
```diff
@@ -1,50 +0,0 @@
...
-
-**February 2014** - Holiday Extras’s ‘mobile moment’ has not happened just yet but with it around the corner, we took a trip to Appsworld in San Francisco https://www.apps-world.net/northamerica/.  This event demonstrated just how fast moving and diverse the mobile app space is and just how much there is to learn, or to quote @lukew ‘unlearn’.  A huge event, too much to cover by just the three of us who went - later in the year some of our developers went to the London event, so we’ve covered most of the content.  The highlight has to have been listening to Steve Wozniak being interviewed - two points really resonated as a Product Owner - design is important - how something looks is very important - looks can convince people to buy it, and this applies to both the company and the product.  And his other point is that is real important to know what features not to include - the ability to say No is critical.
```
Comment (alannamooney) [inline _drafts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Should "and his other point is that is real important" be "and his other point is that it's real important"?

File: _drafts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 3]
```diff
@@ -1,50 +0,0 @@
...
-
-ps.  San Francisco is an amazing place, I fell in love with it.  If you get a chance, go!
-
-
-**March 2014** - Found a neat (new-to-me) lean product development conference - LeanDayLondon https://leanday.splashthat.com/ (found through the equally resourceful site for product owners http://www.mindtheproduct.com/).  Really recommend this 2-day conference - high quality talks and I loved the fact that the content was also aimed at applying lean in large, established companies and not merely start-ups alone.  Loved listening to Dr Tendayi Viki talk about the Ambidextrous Organisation and the biggest challenge for large organisations trying to innovate is their size - they have to keep their cash cow going but innovate at the same time (they need to develop new cows).
```
Comment (alannamooney) [inline _drafts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Should "Ambidextrous Organisation and the biggest challenge for large organisations" be "Ambidextrous Organisation and how the biggest challenge for large organisations"

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md [hunk 4]
```diff
@@ -1,22 +1,12 @@
...
-layout: post
-published: false
-title: "The journey to small, cross-functional, Agile teams"
-comments: "true"
+title: The journey to small, cross-functional, Agile teams
+categories: business
 author: Anna Divers
-categories: Business
-tags: "Organisational structure, Agile, Scrum"
+tags: Organisational-Structure Agile Scrum
```
Comment (alannamooney) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: do the tags need to be separated by commas?

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md [hunk 5]
```diff
@@ -52,7 +42,7 @@ Here's what we learned along the way.
 
 **Product Owners need quality time and sole focus.  They also need to be available**  Once again we didn’t have spare Product Owners lying around, but recognised the role needed to be pretty senior due to the level of accountability and decision-making it would have.  So our Associate Directors stepped into the role for the first year - dual-running however is difficult so we have now looked at ways in which we can fill the product owner roles to enable them to step out of the detail.
```
Comment (alannamooney) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: Collaboration is key
“By involving members of the team meant we had ambassadors” would read better without “By” at the start. 

Communicating to the rest of the business
“Whilst comms in the web team itself was good” should be “Whilst comms in the web team itself were good"
“It is probably fair to say” - it doesn’t need a capital letter as it’s after a comma. 

Conclusion
“Supportive of each other along the way” would read better as “and are supportive of each other along the way”.

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md (hunk 4 above)
Comment (jodiedoubleday) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: nope preferably they won't be

**Comment by stuartstobie** [comment]
Embarrassing to say I love proof reading. I wont be a reviewer but I'll have a look anyway!

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md [hunk 6]
```diff
@@ -24,11 +14,11 @@ Here's what we learned along the way.
 
 **It takes a lot of planning.**  This time a year ago (January) we started the planning, it wasn’t until mid-May that we made the move.   So many factors to consider - what pods were needed, what roles were needed in each pod, how would we measure the success of the pods, how would we transition existing work, how would we allocate work and finally who should go in each pod?
```
Comment (stuartstobie) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: *There were* so many factors to consider...

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md [hunk 7]
```diff
@@ -24,11 +14,11 @@ Here's what we learned along the way.
...
 
-**Collaboration is key.**  A group of senior managers worked together and involved a number of the team, together with our People Team in evolving the plan.  By involving members of the team meant we had ambassadors when it came to the final, public comms.  We were also able to predict and address some of the questions which came up in advance.
+**Collaboration is key.**  A group of senior managers worked together and involved a number of the team, together with our People Team in evolving the plan.  Involving members of the team meant we had ambassadors when it came to the final, public comms.  We were also able to predict and address some of the questions which came up in advance.
```
Comment (stuartstobie) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: A lot of 'togethers' and 'involvings'  here for a hard to read sentence... hows about: 'A group of senior managers worked together *with* a number of the Web Team and our People Team to evolve the plan.'

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md [hunk 8]
```diff
@@ -24,11 +14,11 @@ Here's what we learned along the way.
...
 
 **The right mix is important** - the roles, personalities, level of experience and skill-set.  It’s important to get this right so that the pods have the greatest chance of early success.  
 
-**Communicating to the rest of the business.**  Whilst comms in the web team itself was good and well received, It is probably fair to say we didn’t do enough of this to the wider business.  We had sent communication out and made a video on pods, but there was clearly more needed to understand the changes and how it impacted them.   More than one person thought the pods were the little red meeting booths we have peppered around the business!  Oops!
+**Communicating to the rest of the business.**  Whilst comms in the web team itself were good and well received, it is probably fair to say we didn’t do enough of this to the wider business.  We had sent communication out and made a video on pods, but there was clearly more needed to understand the changes and how it impacted them.   More than one person thought the pods were the little red meeting booths we have peppered around the business!  Oops!
```
Comment (stuartstobie) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: We had sent communication*s* out and made a video on pods, but ~~there was~~ clearly more *was* needed *for everyone to fully* understand the changes and how it *would* impact~~ed~~ them.

File: _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md [hunk 9]
```diff
@@ -24,11 +14,11 @@ Here's what we learned along the way.
...
 
 **Getting the balance right.**  We had geared our pods around our strategic themes and whilst we did have a trading pod from day one, we under-resourced this and had to address this later on in the year.
```
Comment (stuartstobie) [inline _posts/2015-01-12-the-journey-to-small-cross-functional-agile-teams.md]: Repetitive: "we under-resourced this and had to address ~~this~~ *the problem* later on in the year."

**Comment by stuartstobie** [comment]
*Disciplined* get-togethers are still important.
And, personally, I prefer the phrase 'fly solo' to 'run solo' regarding the less experienced teams.

Otherwise reads nicely and interesting to hear your (Anna's) perspective on it a someone who came in after pods had taken effect.

**Pull request merged**